cmd/kdk/init.go

```diff
@@ -37,7 +37,7 @@ func init() {
 	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.Name, "name", "n", "kdk", "KDK Name")
 	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.Port, "port", "p", "2022", "KDK Port")
 	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.ImageRepository, "image-repository", "r", "ciscosso/kdk", "KDK Image Repository")
-	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.ImageTag, "image-tag", "t", "debian-latest", "KDK Image Tag")
+	initCmd.Flags().StringVarP(&CurrentKdkEnvConfig.ConfigFile.AppConfig.ImageTag, "image-tag", "t", "latest", "KDK Image Tag")
```

Comment:
NOTE: In the future, the user should not provide the image-tag except as an override to kdk up.

Instead, the image tag should not be embedded in the config file.  It should be inferred by the kdk version itself.  The kdk bin should work in lock step with the same version of the docker image.